build.gradle.kts

```diff
@@ -21,31 +21,33 @@ configurations.matching { it.name == "detekt" }.all {
   }
 }
 
-val springDocVersion = "1.7.0"
 val sentryVersion = "7.3.0"
 
 dependencies {
   implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.7.3")
   implementation("org.springframework.boot:spring-boot-starter-webflux")
   implementation("org.springframework.boot:spring-boot-starter-data-jpa")
   implementation("org.springframework.retry:spring-retry")
-  implementation("com.vladmihalcea:hibernate-types-55:2.21.1")
+  implementation("org.springframework.integration:spring-integration-jms")
+  implementation("io.hypersistence:hypersistence-utils-hibernate-63:3.7.0")
   implementation("org.locationtech.jts:jts-core:1.19.0")
-  implementation("org.hibernate:hibernate-spatial")
+  implementation("org.hibernate:hibernate-spatial:6.4.4.Final")
   implementation("org.flywaydb:flyway-core")
   implementation("org.springframework.boot:spring-boot-starter-data-redis")
   implementation("org.springframework.boot:spring-boot-starter-cache")
   implementation("com.github.ben-manes.caffeine:caffeine")
   implementation("com.google.guava:guava:33.0.0-jre")
 
-  runtimeOnly("org.postgresql:postgresql:42.7.1")
+  implementation("org.postgresql:postgresql:42.7.3")
```

Comment:
Project does not build if org.postgres dependency is runtime only, there are compilation errors related to:

`import org.postgresql.ds.PGSimpleDataSource`

in [DbExtension.kt](https://github.com/ministryofjustice/hmpps-approved-premises-api/blob/53b8fd60a16f70dee8484c3988e9550a3f007b27/src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/util/DbExtension.kt#L7C1-L8C1).

Reply:
Ah that's interesting. I suspect 'old' spring was pulling it in at implementation() level in the background otherwise DbExtension wouldn't have compiled which is odd! Happy to leave as is